public/i18n.js

```diff
@@ -110,7 +115,10 @@ const locales = {
     "ftm-site-launch-info": "This site was launched on <a href=\"https://twitter.com/arrkay/status/1240704610186092546\">March 19th</a> by <a href=\"https://www.linkedin.com/in/rachelpopkin/\">Rachel Popkin</a> and has been improved by many fantastic volunteers. Many of us are tech workers in Seattle. We are advised by a group of doctors and nurses around the country.",
     "ftm-anonymous-volunteers": "and others who are anonymous",
     "ftm-ack-data-sources": "This site also includes data from <a href=\"https://tinyurl.com/hospitals-needing-ppe\">https://tinyurl.com/hospitals-needing-ppe</a>, a project by:",
-    "ftm-project-volunteers": "Our volunteers include:"
+    "ftm-project-volunteers": "Our volunteers include:",
+    "ftm-maps-search-placeholder": "Enter ZIP code or address, e.g. 94070",
+    "ftm-maps-search-use-location": "Use my location",
+    "ftm-maps-search-reset": "Reset map",
```

Comment:
do we need entries for each localization to avoid breakage?

Reply:
Feel free to do later in a new pr. That file keeps changing 😞

Reply:
sounds good.